bolt-modules/boltlib/lib/puppet/functions/set_feature.rb

```diff
@@ -41,7 +39,5 @@ def set_feature(target, feature, value = true)
     executor&.report_function_call('set_feature')
 
     inventory.set_feature(target, feature, value)
-
-    target
```

Comment:
I dont think we meant to return a target here? Any reason to?

Reply:
I think we may have intended it so so that it could be chained.